pipeline_games/load_games.py

```diff
@@ -0,0 +1,256 @@
+"""Script for loading to database"""
+from os import environ
+from dotenv import load_dotenv
+import pandas as pd
+from psycopg2 import connect, DatabaseError, sql
+from psycopg2.extras import RealDictCursor, execute_batch
+
+
+def get_db_connection(config):
+    """Connect to the database with game data"""
+    try:
+        return connect(
+            user=config['DATABASE_USERNAME'],
+            password=config['DATABASE_PASSWORD'],
+            host=config['DATABASE_IP'],
+            port=config['DATABASE_PORT'],
+            database=config['DATABASE_NAME'])
+    except ValueError:
+        return "Error connecting to database."
+
+
+def execute_batch_columns(conn, data: pd.DataFrame, table: str, column: str, page_size=100):
+    """Using psycopg2.extras.execute_batch() to insert the dataframe"""
+    tuples = tuples = list(zip(data.unique()))
+    cols = column
+    query = """INSERT INTO %s(%s) VALUES(%%s)""" % (table, cols)
+    with conn.cursor(cursor_factory=RealDictCursor) as cur:
+        try:
+            execute_batch(cur, query, tuples, page_size)
+            conn.commit()
+            cur.close()
+            print("execute_batch() done")
+        except (Exception, DatabaseError) as error:
+            print(f"Error: {error}")
+            conn.rollback()
```

Comment:
Curiosity Questions: What does` conn.rollback()` do? 

I assume it reverts any changes that the may have happened in the database during this functions execution?

Reply:
yea exactly that -- it stops changes from being made by accident onto the database!